diff --git a/builds/unix/freetype2.in b/builds/unix/freetype2.in
index c4dfda4..97f256e 100644
--- a/builds/unix/freetype2.in
+++ b/builds/unix/freetype2.in
@@ -7,7 +7,7 @@ Name: FreeType 2
 URL: http://freetype.org
 Description: A free, high-quality, and portable font engine.
 Version: %ft_version%
-Requires:
+Requires: %REQUIRES_PRIVATE%
 Requires.private: %REQUIRES_PRIVATE%
 Libs: -L${libdir} -lfreetype
 Libs.private: %LIBS_PRIVATE%
