--- a/mpn/x86_64/k8/redc_1.asm	2014-03-25 15:37:55.000000000 +0100
+++ a/mpn/x86_64/k8/redc_1.asm	2015-09-11 14:30:57.000000000 +0200
@@ -114,7 +114,7 @@
 
 	JUMPTABSECT
 	ALIGN(8)
-L(tab):	JMPENT(	L(0m4), L(tab))
+L(tab):	JMPENT(	L(0), L(tab))
 	JMPENT(	L(1), L(tab))
 	JMPENT(	L(2), L(tab))
 	JMPENT(	L(3), L(tab))
@@ -397,6 +397,7 @@
 
 
 	ALIGN(16)
+L(0):
 L(0m4):
 L(lo0):	mov	(mp,nneg,8), %rax
 	mov	nneg, i
