diff --git a/common/wscript b/common/wscript
index 4302b1e..c39aec1 100644
--- a/common/wscript
+++ b/common/wscript
@@ -380,7 +380,7 @@ def build(bld):
          process = create_jack_process_obj(bld, 'audioadapter', audio_adapter_sources, serverlib)
          process.use = 'SAMPLERATE'
 
-    if bld.env['BUILD_ADAPTER'] and bld.env['IS_WINDOWS']:
+    if bld.env['BUILD_ADAPTER'] and bld.env['HAVE_PORTAUDIO']:
          audio_adapter_sources += ['../windows/portaudio/JackPortAudioAdapter.cpp', '../windows/portaudio/JackPortAudioDevices.cpp']
          process = create_jack_process_obj(bld, 'audioadapter', audio_adapter_sources, serverlib)
          process.use += ['SAMPLERATE', 'PORTAUDIO']
diff --git a/windows/wscript b/windows/wscript
index ea4dd3d..f00d66e 100644
--- a/windows/wscript
+++ b/windows/wscript
@@ -2,7 +2,7 @@
 # encoding: utf-8
 
 def configure(conf):
-    conf.check_cfg(package='portaudio-2.0', uselib_store='PORTAUDIO', atleast_version='19', args='--cflags --libs')
+    conf.check_cfg(package='portaudio-2.0', uselib_store='PORTAUDIO', atleast_version='19', args='--cflags --libs', mandatory=False)
     conf.env['BUILD_DRIVER_PORTAUDIO'] = conf.is_defined('HAVE_PORTAUDIO')
 
 def create_jack_driver_obj(bld, target, sources, uselib = None):
