diff --git a/common/wscript b/common/wscript
index 4302b1e..53d453d 100644
--- a/common/wscript
+++ b/common/wscript
@@ -7,11 +7,12 @@ import os
 
 def configure(conf):
     conf.check_cc(header_name='samplerate.h', define_name="HAVE_SAMPLERATE")
+    conf.check_cc(header_name='pthread.h', define_name="HAVE_PTHREAD", mandatory=False)
    
     if conf.is_defined('HAVE_SAMPLERATE'):
         conf.env['LIB_SAMPLERATE'] = ['samplerate']
 
-    conf.env['BUILD_NETLIB'] = conf.is_defined('HAVE_SAMPLERATE')
+    conf.env['BUILD_NETLIB'] = conf.is_defined('HAVE_SAMPLERATE') and conf.is_defined('HAVE_PTHREAD')
     conf.env['BUILD_ADAPTER'] = conf.is_defined('HAVE_SAMPLERATE')
 
     if conf.env['IS_WINDOWS']:
@@ -82,7 +83,7 @@ def build(bld):
         includes.append('..')
     else:
         includes.append('../..')
-    uselib = ["PTHREAD", "CELT", "OPUS"]
+    uselib = ["CELT", "OPUS"]
 
     if bld.env['IS_LINUX']:
         common_libsources += [
