diff --git a/common/wscript b/common/wscript
index 4302b1e..53d453d 100644
--- a/common/wscript
+++ b/common/wscript
@@ -380,7 +381,7 @@ def build(bld):
          process = create_jack_process_obj(bld, 'audioadapter', audio_adapter_sources, serverlib)
          process.use = 'SAMPLERATE'
 
-    if bld.env['BUILD_ADAPTER'] and bld.env['IS_WINDOWS']:
+    if bld.env['BUILD_ADAPTER'] and bld.env['HAVE_PORTAUDIO']:
          audio_adapter_sources += ['../windows/portaudio/JackPortAudioAdapter.cpp', '../windows/portaudio/JackPortAudioDevices.cpp']
          process = create_jack_process_obj(bld, 'audioadapter', audio_adapter_sources, serverlib)
          process.use += ['SAMPLERATE', 'PORTAUDIO']
diff --git a/windows/wscript b/windows/wscript
index fdae9bb..069fd53 100644
--- a/windows/wscript
+++ b/windows/wscript
@@ -6,7 +6,7 @@ import re
 import os
 
 def configure(conf):
-    conf.check_cc(function_name='Pa_GetVersion', header_name='portaudio.h', lib='portaudio', uselib_store="PORTAUDIO", define_name='HAVE_PORTAUDIO')
+    conf.check_cc(function_name='Pa_GetVersion', header_name='portaudio.h', lib='portaudio', uselib_store="PORTAUDIO", define_name='HAVE_PORTAUDIO', mandatory=False)
     conf.env['BUILD_DRIVER_PORTAUDIO'] = conf.is_defined('HAVE_PORTAUDIO')
 
 def build(bld):
