diff --git a/common/wscript b/common/wscript
index 4302b1e..53d453d 100644
--- a/common/wscript
+++ b/common/wscript
@@ -6,7 +6,7 @@ import re
 import os
 
 def configure(conf):
-    conf.check_cc(header_name='samplerate.h', define_name="HAVE_SAMPLERATE")
+    conf.check_cc(header_name='samplerate.h', define_name="HAVE_SAMPLERATE", mandatory=False)
    
     if conf.is_defined('HAVE_SAMPLERATE'):
         conf.env['LIB_SAMPLERATE'] = ['samplerate']
diff --git a/wscript b/wscript
index aef4bd8..98d76ca 100644
--- a/wscript
+++ b/wscript
@@ -166,7 +166,7 @@ def configure(conf):
         if conf.env['BUILD_JACKDBUS'] != True:
             conf.fatal('jackdbus was explicitly requested but cannot be built')
 
-    conf.check_cc(header_name='samplerate.h', define_name="HAVE_SAMPLERATE")
+    conf.check_cc(header_name='samplerate.h', define_name="HAVE_SAMPLERATE", mandatory=False)
 
     if conf.is_defined('HAVE_SAMPLERATE'):
         conf.env['LIB_SAMPLERATE'] = ['samplerate']
