diff --git a/wscript b/wscript
index aef4bd8..06cf2ba 100644
--- a/wscript
+++ b/wscript
@@ -171,7 +171,7 @@ def configure(conf):
     if conf.is_defined('HAVE_SAMPLERATE'):
         conf.env['LIB_SAMPLERATE'] = ['samplerate']
 
-    conf.sub_config('example-clients')
+    #conf.sub_config('example-clients')
 
     if conf.check_cfg(package='celt', atleast_version='0.11.0', args='--cflags --libs', mandatory=False):
         conf.define('HAVE_CELT', 1)
@@ -429,7 +429,7 @@ def build(bld):
 
     if bld.env['IS_WINDOWS']:
         bld.add_subdirs('windows')
-        bld.add_subdirs('example-clients')
+        #bld.add_subdirs('example-clients')
         #bld.add_subdirs('tests')
 
     if bld.env['BUILD_DOXYGEN_DOCS'] == True:
