diff -r b35a5d8f012b source/CMakeLists.txt
--- a/source/CMakeLists.txt	Sun May 18 15:02:27 2014 +0900
+++ b/source/CMakeLists.txt	Sat Oct 31 21:36:55 2015 +0100
@@ -12,7 +12,7 @@
 endif()
 
 project (x265)
-cmake_minimum_required (VERSION 2.8.8) # OBJECT libraries require 2.8.8
+cmake_minimum_required (VERSION 2.8.7) # OBJECT libraries require 2.8.8
 include(CheckIncludeFiles)
 include(CheckFunctionExists)
 include(CheckSymbolExists)
